Sources/DatadogCrashReporting/PLCrashReporterIntegration/DDCrashReportExporter.swift

```diff
@@ -150,7 +150,7 @@ internal struct DDCrashReportExporter {
 
             return DDCrashReport.BinaryImage(
                 libraryName: image.imageName,
-                uuid: image.uuid,
+                uuid: image.uuid ?? unavailable,
```

Comment:
It is still relevant to report binary images with no UUID, like we do for missing architecture. Without uuid nor arch, symbolication won't be possible, but this will give more info to the crash report and help us identify missing data.

Reply:
Double checking - this will be possible to differentiate one `???` from another `???` by exploring the content of `binary_images[]`, right?